main.go

```diff
@@ -108,6 +110,12 @@ func main() {
 
 	if numChanges > 0 && !yes {
 		if len(deletes) > 0 {
+			if leaveunknown {
```

Comment:
This won't fire if the user supplied `-yes`.

Reply:
I would suggest inserting something like this between `deletes := deleteCandidates.Difference(updates, Updatable)` and  `numChanges := len(updates) + len(adds) + len(deletes)`:

```go
if len(deletes) > 0 && leaveunknown {
    fmt.Fprintf(stdout, "%d unknown records left untouched\n", len(deletes))
    deletes = deletes[:0]
}
```